tests/stack/stack_test.go

```diff
@@ -500,12 +500,6 @@ func TestStackRenameAfterCreateServiceBackend(t *testing.T) {
 	e.RunCommandExpectError("pulumi", "stack", "rename", "fakeorg/"+stackRenameBase)
 
 	// Next perform a legal rename. This should work.
-	e.RunCommand("pulumi", "stack", "rename", stackRenameBase)
-	stdoutXyz1, _ := e.RunCommand("pulumi", "config", "get", "xyz")
-	assert.Equal(t, "abc", strings.Trim(stdoutXyz1, "\r\n"))
-
-	// Now perform another legal rename, this time explicitly specifying the
-	// "organization" for the stack (which should match the default).
```

Comment:
Callout this is the tradeoff of using individual org instead of default org -- Compare with the diff in https://github.com/pulumi/pulumi/pull/19249/files#diff-6dfc4008d710f3752d26ea7d66b640f4f03485610dcb4f4570983de2d2529219

Happy to go with either option!